### ISSUE TITLE
 - fungio/google-map-bundle 4.3.x-dev requires symfony/framework-bundle 4.* -> no matching package found.

### ISSUE THREAD

**Issue by augustomurri** [fungio/FungioGoogleMapBundle#6]
Is this bundle compatible with symfony5 ?

**Comment by fungio** [comment]
I guess you got to change the composer.json to make it compatible, otherwise it should be working just fine. Give it a try ;)

**Comment by augustomurri** [comment]
I tried to change 

`        "symfony/framework-bundle": "5.*",`
to
`        "symfony/framework-bundle": "4.*",`

But it throw many other dependency errors :-/

```
  Problem 1
    - The requested package symfony/framework-bundle 4.10.0.* exists as symfony/framework-bundle[5.0.x-dev, v5.0.0, v5.0.0-BETA1, v5.0.0-BETA2, v5.0.0-RC1, v5.0.1, v5.0.10, v5.0.2, v5.0.3, v5.0.4, v5.0.5, v5.0.6, v5.0.7, v5.0.8, v5.
0.9] but these are rejected by your constraint.
  Problem 2
    - The requested package symfony/framework-bundle (locked at v5.0.7, required as 4.10.0.*) is satisfiable by symfony/framework-bundle[v5.0.7] but these conflict with your requirements or minimum-stability.
  Problem 3
    - Installation request for fungio/google-map-bundle ^4.3@dev -> satisfiable by fungio/google-map-bundle[4.3.x-dev].
    - fungio/google-map-bundle 4.3.x-dev requires symfony/framework-bundle 4.* -> no matching package found.
  Problem 4
    - doctrine/doctrine-bundle 2.0.8 requires symfony/framework-bundle ^3.4.30|^4.3.3|^5.0 -> satisfiable by symfony/framework-bundle[5.0.x-dev, v5.0.0, v5.0.0-BETA1, v5.0.0-BETA2, v5.0.0-RC1, v5.0.1, v5.0.10, v5.0.2, v5.0.3, v5.0.4
, v5.0.5, v5.0.6, v5.0.7, v5.0.8, v5.0.9] but these conflict with your requirements or minimum-stability.
    - doctrine/doctrine-bundle 2.0.8 requires symfony/framework-bundle ^3.4.30|^4.3.3|^5.0 -> satisfiable by symfony/framework-bundle[5.0.x-dev, v5.0.0, v5.0.0-BETA1, v5.0.0-BETA2, v5.0.0-RC1, v5.0.1, v5.0.10, v5.0.2, v5.0.3, v5.0.4
, v5.0.5, v5.0.6, v5.0.7, v5.0.8, v5.0.9] but these conflict with your requirements or minimum-stability.
    - doctrine/doctrine-bundle 2.0.8 requires symfony/framework-bundle ^3.4.30|^4.3.3|^5.0 -> satisfiable by symfony/framework-bundle[5.0.x-dev, v5.0.0, v5.0.0-BETA1, v5.0.0-BETA2, v5.0.0-RC1, v5.0.1, v5.0.10, v5.0.2, v5.0.3, v5.0.4
, v5.0.5, v5.0.6, v5.0.7, v5.0.8, v5.0.9] but these conflict with your requirements or minimum-stability.
    - Installation request for doctrine/doctrine-bundle (locked at 2.0.8) -> satisfiable by doctrine/doctrine-bundle[2.0.8].

```

**Comment by fungio** [comment]
Fixed with you Pull request #7
